karma.conf.js

```diff
@@ -0,0 +1,69 @@
+// Karma configuration
+// Generated on Sat Apr 01 2017 01:03:22 GMT-0500 (CDT)
+
+module.exports = function(config) {
+  config.set({
+
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '',
+
+
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: ['mocha'],
+
+
+    // list of files / patterns to load in the browser
+    files: [
+      'src/**/*.spec.js',
```

Comment:
As far as locations for these tests, do we want to house the test for each pattern inside the same directory like all the javascript itself, or do we want a designated `test` directory for this?  

So for example, this file exists: 
`styleguide/derek/global-components/mainnav/mainnav.js`
Would we just want the spec right next to it?
`styleguide/derek/global-components/mainnav/mainnav.spec.js`